apps/prairielearn/src/pages/administratorSettings/administratorSettings.ts

```diff
@@ -1,11 +1,12 @@
 // @ts-check
```

Comment:
You can remove this comment now that this file has a `.ts` extension.

```suggestion
```

Reply:
Resolved in commit 7086e1f.